flux/dev/model/model.py

```diff
@@ -0,0 +1,83 @@
+import base64
+import logging
+import math
+import random
+import subprocess
+import time
+from io import BytesIO
+
+import numpy as np
+import torch
+from diffusers import FluxPipeline
+from PIL import Image
+
+logging.basicConfig(level=logging.INFO)
+MAX_SEED = np.iinfo(np.int32).max
+
+
+class Model:
+    def __init__(self, **kwargs):
+        self._secrets = kwargs["secrets"]
+        self.model_name = kwargs["config"]["model_metadata"]["repo_id"]
+        self.hf_access_token = self._secrets["hf_access_token"]
+        self.pipe = None
+
+    def load(self):
+        self.pipe = FluxPipeline.from_pretrained(
+            self.model_name, torch_dtype=torch.bfloat16, token=self.hf_access_token
```

Comment:
nit: `torch_dtype` auto might be faster?

Reply:
auto is supposed to infer the dtype from the weights, but in their calling code, flux explicitly define bfloat16, so I used that.
https://huggingface.co/black-forest-labs/FLUX.1-dev